File: python/nb.py
import sys

print('What is your first name?  ')
fName = sys.stdin.readline()
print('What is your last name?  ')
lName = sys.stdin.readline()

if fName == 'Linus\n' and lName == 'Zhang\n':
  print('> 牛逼!')
else:
  print('> 不牛逼!')

File: c++/nb.cpp
#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    
    string firstName;
    string lastName;

    cout << "What is your first name?\n";
    cin >> firstName;
    cout << "What is your last name?\n";
    cin >> lastName;
    
    cout << "> " << ( firstName == "Linus" && lastName == "Zhang" ? "牛逼" : "不牛逼" ) << "\n";
    return 0;
}
